### PR TITLE
Handle ConnectionError

### PR DESCRIPTION
**Pull request by fernandog**
@miigotu hotfix?

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

**Comment by miigotu** [comment]
ConnectionError is undefined.

Anyways, look at this: http://stackoverflow.com/questions/16511337/correct-way-to-try-except-using-python-requests-module

**Comment by fernandog** [comment]
So I cant use 'as e' ?

**Comment by miigotu** [comment]
OFC you can, but the exception you are catching is not imported.....

**Comment by fernandog** [comment]
@miigotu Done.

File: sickbeard/helpers.py [hunk 1]
```diff
@@ -1684,6 +1687,9 @@ def download_file(url, filename, session=None, headers={}):
             except Exception:
                 logger.log(u"Problem setting permissions or writing file to: %s" % filename, logger.WARNING)
 
+    except ConnectionError as e:
+        logger.log(u"No internet available while loading download URL %s Error: %r" % (url, ex(e)))
```
Comment (fernandog) [inline sickbeard/helpers.py]: we already have this some row below so why we still have ConnectionError?

`except requests.exceptions.ConnectionError as e:`

**Comment by fernandog** [comment]
"All checks have passed" - its appearing again

**Comment by fernandog** [comment]
@miigotu did you see I added socket.error?  why the connection error raise even if we have the except for it ?

**Comment by miigotu** [comment]
first, socket.error is undefined.

second, you didnt read past the first post on the SO answer I linked you to.

**Comment by fernandog** [comment]
sorry. I didn't get it

**Pull request merged**